src/container.ts

```diff
@@ -85,8 +74,8 @@ const samlCert = () => {
     "SAML certificate"
   );
 };
-export const SAML_CERT = "samlCert";
-const SAML_CERT_VALUE = samlCert();
```

Comment:
let's minimize the changes, for now keep the var names as before, keep the `_VALUE` suffix

Reply:
The `_VALUE` version of the variables was used only inside the container. Now that the variables are exported, in order to minimize the lines changes across all files, I think that it is better to rename them.